packages/dev/query/package.json

```diff
@@ -1,7 +1,18 @@
 {
   "name": "parcel-query",
```

Comment:
Should `parcel-query` be published under the `@parcel` namespace?

Reply:
Yeah, might be a good idea for consistency